filament/backend/src/vulkan/VulkanPipelineCache.h

```diff
@@ -197,6 +218,7 @@ class VulkanPipelineCache : public CommandBufferObserver {
         VkVertexInputAttributeDescription vertexAttributes[VERTEX_ATTRIBUTE_COUNT]; // 256 bytes
         VkVertexInputBindingDescription vertexBuffers[VERTEX_ATTRIBUTE_COUNT];      // 192 bytes
         uint32_t padding1;                                                          // 4 bytes
+        LayoutBundle* layout = nullptr;              // 8 bytes
```

Comment:
I think this field might not be necessary because the shader module is already in the key, and the layout is implied by the shader.  What do you think?

Reply:
Yes shaders will be enough. It's removed.